src/asciidocUtils.ts

```diff
@@ -24,11 +24,10 @@ export function getADParser() {
 }
 
 export function parseADtoHTML(content: string) {
-    const html = asciidoctor.convert(content, { 'safe': 'server', 
-        'attributes': { 
-            'showtitle': true, 
-            'icons': 'font', 
-            'stylesheet': 'webview.css' 
+    const html = asciidoctor.convert(content, { 'safe': 'server',
```

Comment:
We're actually providing the stylesheets via the code at https://github.com/redhat-developer/vscode-didact/blob/cf5ef124038fbaf94624b203ca7244fb896b0b74/src/didactWebView.ts#L256 so this should not be necessary.

Reply:
Do you want to revert the suggested change? `webview.css` does not even exist what is the purpose to pass it to the renderer?

Reply:
that's an EXCELLENT point. forgot about that!